src/styles/blocks/card.css

```diff
@@ -0,0 +1,123 @@
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 0 16px;
+}
+
+/* CARD ICON */
+
+.card__icon {
+  width: 160px;
+  height: 134px;
+
+  margin: 0 19px 36px;
+
+  /* margin: 0 19px 36px; */
+}
+
+/* CARD INFO */
+
+.card__info {
+  display: flex;
+  flex-direction: column;
+  margin: 0 16px;
+}
+
+.card__info-title {
+  font-size: var(--medium-size-text);
+  font-weight: 500;
+  color: var(--emphasized-text-color);
+
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__info-productCode {
+  font-size: var(--small-size-text);
+  font-weight: var(--light-weight);
+  color: var(--lightened-text-color);
+
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+/* CARD REVIEW */
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  line-height: 14px;
+
+  margin-bottom: 24px;
+}
+
+.card__review-mark {
+  font-size: var(--small-size-text);
+  font-weight: var(--light-weight);
+
+  color: var(--emphasized-text-color);
+}
+
+/* CARD PRICE */
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: var(--medium-size-text);
+  font-weight: var(--light-weight);
+
+  color: var(--lightened-text-color);
+}
+
+.card__price-sum {
+  font-size: 16px;
+  font-weight: var(--bold-weight);
+
+  color: var(--emphasized-text-color);
+  line-height: 18px;
+  /* margin-bottom: 16px; */
+}
+
+/* CARD BUTTON  */
+
+.card__button-buy {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  font-size: 14px;
+  font-weight: var(--bold-weight);
+
+  color: #fff;
+  background-color: var(--button-color);
+
+  border-radius: 5px;
+
+  width: 166px;
+  height: 40px;
+
+  transition: ease;
+  -webkit-transition: ease 0.3s;
+}
+
+.card__button-buy:hover {
+  background-color: #fff;
+  border: 1px solid var(--button-color);
```

Comment:
You should add transparent or blue border to `.card__button-buy` for smoother animation and reset only the color of this border in `:hover`